woocommerce/woocommerce-android
# Quick win: Remove custom `android_update_metadata_source` action

### Description
Removes a custom action in favor of calls in the `Fastfile` itself. The intent is to make the code leaner and to remove a level of indirection to understand what happens.

See also:

- https://github.com/wordpress-mobile/WordPress-Android/pull/16404
- paaHJt-3fL-p2

### Testing instructions

I tested this using a test PR: https://github.com/woocommerce/woocommerce-android/pull/6381

1. Update the release notes, just to verify the changes take place
2. Run `bundle exec fastlane update_appstore_strings version:9.0` after the initial refactor (the first 3 commits)
3. Reverted the commit (so I could do the same with the new code and compare)
4. Apply the remaining commits
5. Run `bundle exec fastlane update_appstore_strings version:9.0` again and ensure the diff is the same as before my change

To verify: [Before change diff](https://github.com/woocommerce/woocommerce-android/pull/6381/commits/653add70c3bffc6f9433a87b914b102f526c54c8), [after change diff](https://github.com/woocommerce/woocommerce-android/pull/6381/commits/ab24f4f563fe5e58e6136d5047a7868310c0c82f).

---
